src/tests/nativeaot/SmokeTests/DotnetJs/wwwroot/package.json

```diff
@@ -0,0 +1,3 @@
+{
+    "type": "module"
+}
```

Comment:
Yes, that's because of use of ES6 modules. `.mjs` is alternative solution. I'm not if that requires to have all ES6 modules as `.mjs` (including dotnet.js)

Reply:
It doesn't requite that from other than top level module. But older versions of nodejs don't respect `.mjs` well.